scripts/quickstart.sh

```diff
@@ -5,6 +5,11 @@
 
 trap 'kill 0' SIGTERM
 
+export MINIO_EXECUTABLE="minio"
```

Comment:
Why not just variable without `export`?

Reply:
I just want to use variables dynamically. I didn't care to define it as an environment variable.

Reply:
So can we remove export then? (:

Reply:
Sure :)